# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/simdutf/hotfix-ppc-pre8.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- simdutf-7.7.1/include/simdutf/portability.h.vanilla	2026-01-07 19:09:59.487842309 +0100
+++ simdutf-7.7.1/include/simdutf/portability.h	2026-01-07 19:13:13.334942175 +0100
@@ -109,7 +109,7 @@
 #elif defined(__aarch64__) || defined(_M_ARM64) || defined(_M_ARM64EC)
   #define SIMDUTF_IS_ARM64 1
 #elif defined(__PPC64__) || defined(_M_PPC64)
-  #if defined(__VEC__) && defined(__ALTIVEC__)
+  #if defined(__VEC__) && defined(__ALTIVEC__) && defined(__POWER8_VECTOR__)
     #define SIMDUTF_IS_PPC64 1
   #endif
 #elif defined(__s390__)
